### PR TITLE
MARP-620 Update release drafter workflow with multiple options

### PR DESCRIPTION
**Pull request by ndkhanh-axonivy**
I would like to update release drafter workflow which will:
1. Provide the ability to manually create a pre-release note and create the release note for old tag
2. Auto label PRs if the target branches contain some keywords.

### CODE REVIEW

**Comment by ndkhanh-axonivy** [comment]
Hi @ivy-rew , could you please check this PR which related to the Release Drafter, for more detail please check the PR description.

Thank you!

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
-    branches:
-      - master
-  # pull_request event is required only for autolabeler
-  pull_request:
-    # Only following types are handled by the action, but one can default to all as well
-    types: [opened, reopened, synchronize]
-  # pull_request_target event is required for autolabeler to support PRs from forks
-  # pull_request_target:
-  #   types: [opened, reopened, synchronize]
+  workflow_call:
```
Comment (ivy-rew) [inline .github/workflows/release-drafter.yml:4]: ok to have this pipeline invokable for other repos; but now it seems as if we have no trigger for the releases of these 'github-workflows' itself. 
I'd add another pipeline for this ... just using this template as everyone else does.

Comment (ndkhanh-axonivy) [reply .github/workflows/release-drafter.yml:4]: Hi @ivy-rew , I just created a new pipeline self-release-drafter to create the release note for github-workflows project. Please help me to re-check it. Thanks!

File: .github/release-drafter.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name-template: 'v$RESOLVED_VERSION 🧪'
```
Comment (ivy-rew) [inline .github/release-drafter.yml:1]: not a very important thing, but could we maintain our own 'release-drafter-yml' for the self pipeline? would be a good sample to copy from for those that our looking for an own template.

Comment (ndkhanh-axonivy) [reply .github/release-drafter.yml:1]: Hi @ivy-rew , I recently add another template for self pipeline - just make it same as previous version of release-drafter.yaml here. Please help me to check it.

**Review by ivy-rew — APPROVED** [review]

File: .github/workflows/self-release-drafter.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+jobs:
+  update_release_draft:
+    permissions:
+      contents: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: release-drafter/release-drafter@v6
+        with:
+          config-name: self-release-drafter.yml
```
Comment (ivy-rew) [inline .github/workflows/self-release-drafter.yml:22]: :checkered_flag:

**Review by nqhoan-axonivy — APPROVED** [review]
LGTM

**Pull request merged**